### PR TITLE
v0.11.5

### PR DESCRIPTION
**Pull request by m4dcoder**

### CODE REVIEW

**Comment by manasdk** [comment]
I think we are good here. I don't see much point in waiting for travis.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+
+* Fix a bug where ``end_timestamp`` is not captured for Mistral workflow executions (bug-fix)
+* Fix a bug where the CLI failed to display Mistral workflow that errored (bug-fix)
+* Fix a bug where the published variables is not captured in the Mistral workflow result (bug-fix)
+
 0.11.4 - June 30, 2015
 ----------------------
 
-* Remove unncecssary rule notifu_hubot from core.
+* Remove unnecessary rule notify_hubot from core.
```
Comment (manasdk) [inline CHANGELOG.rst]: Thanks!

**Comment by DoriftoShoes** [comment]
LGTM.  I guess we just wait on Travis again...lol.

**Comment by manasdk** [comment]
+1

**Pull request merged**